employees/models.py

```diff
@@ -6,6 +6,7 @@
 from employees.common.validators import MaxDecimalValueValidator
 from managers.models import Project
 from users.models import CustomUser
+from markdown_deux import markdown
```

Comment:
This line should be moved elsewhere.
There's a strict order for package imports, it goes like this:
1. Standard library imports
2. Third-party packages imports
3. Custom packages imports

Each of these groups is also separated by line.
This import should be located in the second group, among Django packages imports.

Don't forget about alphabetical order. 😉